specification/riskiq/data-plane/Microsoft.Easm/preview/2022-09-01-preview/easm.json

```diff
@@ -0,0 +1,5550 @@
+          "format": "date-time",
+          "type": "string"
+        },
+        "firstSeen": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "lastSeen": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "orgIds": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "orgNames": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "registrantContacts": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "registrantNames": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "registrantPhones": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "registrarCreatedAt": {
+          "items": {
+            "$ref": "#/definitions/ObservedLong"
+          },
+          "type": "array"
+        },
+        "registrarNames": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "registrarUpdatedAt": {
+          "items": {
+            "$ref": "#/definitions/ObservedLong"
+          },
+          "type": "array"
+        },
+        "registries": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "sources": {
+          "items": {
+            "$ref": "#/definitions/Source"
+          },
+          "type": "array"
+        },
+        "technicalContacts": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "technicalNames": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "technicalOrgs": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        },
+        "technicalPhones": {
+          "items": {
+            "$ref": "#/definitions/ObservedString"
+          },
+          "type": "array"
+        }
+      },
+      "type": "object"
+    },
+    "AssetId": {
+      "description": "The list of excludes used for the disco group runs, aka assets to exclude from the discovery algorithm.",
+      "properties": {
+        "displayName": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        },
+        "kind": {
+          "enum": [
+            "as",
+            "contact",
+            "domain",
+            "host",
+            "ipAddress",
+            "ipBlock",
+            "page",
+            "sslCert"
```

Comment:
These were changed from SCREAMING_SNAKE_CASE to camelCase since 2022-04-01-preview. Was that a correction? Otherwise, this would be an SDK-breaking change since SDKs support multiple versions and some languages do not ship separate models for each version.